scripts/simulate_trade.py

```diff
@@ -189,7 +189,16 @@ def run_model(model, dataloader, factors=None):
         model, normalize = load_model(config, args.ckpt_path) 
 
         use_volume = config.get('use_volume', False)
-        test_transform = DataTransform(is_train=False, use_volume=use_volume)
+        feature_list = config.get('features', ['Timestamp', 'Open', 'High', 'Low', 'Close'])
```

Comment:
default to ohcl, still allow "use_volume" setting for backward compatibility, and allow setting features to "all" which will use all features in the dataset